### PR TITLE
Add some stubs for missing symbols in CoreText

### PR DESCRIPTION
**Pull request by dingari**

### CODE REVIEW

**Review by CuriousTommy — CHANGES_REQUESTED** [review]

File: CoreText/include/CoreText/CTFrame.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#import <CoreText/CoreTextExport.h>
+#import <CoreFoundation/CoreFoundation.h>
+#import <CoreGraphics/CoreGraphics.h>
+
+CF_IMPLICIT_BRIDGING_ENABLED
+
+typedef const struct CF_BRIDGED_TYPE(id) __CTFrame * CTFrameRef;
```
Comment (CuriousTommy) [inline CoreText/include/CoreText/CTFrame.h:7]: I recommend sticking with how [Apple declares `CTFrameRef` on their website](https://developer.apple.com/documentation/coretext/ctframeref/#declaration).

I don't recommend including stuff that doesn't exist on Apple's Documentation website (unless you have a good logical explanation for why it should exist, besides it existing in Apple's proprietary SDK).

**Review by CuriousTommy — CHANGES_REQUESTED** [review]

File: CoreText/include/CoreText/CTFrame.h (hunk 1 above)
Comment (dingari) [reply CoreText/include/CoreText/CTFrame.h:7]: Noted, I just saw that this style already existed in [CTFont.h](https://github.com/darlinghq/darling-cocotron/blob/64aee1830f5e0ce319c9380f9dfd8db7bfdbc12b/CoreText/include/CoreText/CTFont.h#L26), so I figured it was OK.

The declaration from Apple's website won't compile as a drop-in replacement, do you want to change this line to the following?
```
typedef struct __CTFrame* CTFrameRef;
```

Comment (CuriousTommy) [reply CoreText/include/CoreText/CTFrame.h:7]: > Noted, I just saw that this style already existed in [CTFont.h](https://github.com/darlinghq/darling-cocotron/blob/64aee1830f5e0ce319c9380f9dfd8db7bfdbc12b/CoreText/include/CoreText/CTFont.h#L26), so I figured it was OK.

No worries. Matching how the other code is written is normally the right thing to do, but in this case, it not very obvious what `CF_BRIDGED_TYPE(id)` does or why it is needed. I rather play it safe and only add it in when we know why it is needed.

Maybe @ahyattdev had a good reason for adding in `CF_BRIDGED_TYPE(id)`, but I rather play it safe and not include it if it is not documented on Apple's website.

> The declaration from Apple's website won't compile as a drop-in replacement, do you want to change this line to the following?
> 
> ```
> typedef struct __CTFrame* CTFrameRef;
> ```

That's fine. I'm not sure why Apple's website doesn't show it as a pointer...

Comment (dingari) [reply CoreText/include/CoreText/CTFrame.h]: I totally understand :)

File: CoreText/include/CoreText/CTFrameSetter.h [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#import <CoreText/CTFrame.h>
+#import <CoreText/CTTypesetter.h>
+
+CF_IMPLICIT_BRIDGING_ENABLED
+
+typedef const struct CF_BRIDGED_TYPE(id) __CTFramesetter * CTFramesetterRef;
```
Comment (CuriousTommy) [inline CoreText/include/CoreText/CTFrameSetter.h:6]: I forgot to state this earlier, but there are other places where `CF_BRIDGED_TYPE(id)` should be removed as well.

After you are done with that, can you rebase (fixup) your `CF_BRIDGED_TYPE(id)` removal changes into 64aee1830f5e0ce319c9380f9dfd8db7bfdbc12b

File: CoreText/include/CoreText/CTLine.h [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#import <CoreText/CoreTextExport.h>
+#import <CoreGraphics/CoreGraphics.h>
+
+CF_IMPLICIT_BRIDGING_ENABLED
+
+typedef const struct CF_BRIDGED_TYPE(id) __CTLine * CTLineRef;
```
Comment (CuriousTommy) [inline CoreText/include/CoreText/CTLine.h:6]: Same here

File: CoreText/include/CoreText/CTRun.h [hunk 4]
```diff
@@ -0,0 +1,47 @@
+#import <CoreText/CoreTextExport.h>
+#import <CoreGraphics/CoreGraphics.h>
+
+CF_IMPLICIT_BRIDGING_ENABLED
+
+typedef const struct CF_BRIDGED_TYPE(id) __CTRun * CTRunRef;
```
Comment (CuriousTommy) [inline CoreText/include/CoreText/CTRun.h:6]: Same here

File: CoreText/include/CoreText/CTTypesetter.h [hunk 5]
```diff
@@ -0,0 +1,5 @@
+CF_IMPLICIT_BRIDGING_ENABLED
+
+typedef const struct CF_BRIDGED_TYPE(id) __CTTypesetter * CTTypesetterRef;
```
Comment (CuriousTommy) [inline CoreText/include/CoreText/CTTypesetter.h:3]: Same here

**Review by CuriousTommy — CHANGES_REQUESTED** [review]

File: CoreText/include/CoreText/CTFrameSetter.h (hunk 2 above)
Comment (dingari) [reply CoreText/include/CoreText/CTFrameSetter.h]: Sure! Should be fixed now.

Comment (dingari) [reply CoreText/include/CoreText/CTFrameSetter.h]: Hm, I'm just now noticing a couple of other places where I used `CF_BRIDGED_TYPE`. I'll fix them now.

**Pull request merged**